recipes/pyreports/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyreports" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyreports-{{ version }}.tar.gz
+  sha256: 59873c391096609c103232fe7617760810824e81c9bef366b26f1944d05f2acb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - ldap3
+    - mysql-connector-python
+    - nosqlapi
+    - psycopg2
+    - pymssql
+    - python >=3.7
+    - tablib
+
+test:
+  imports:
+    - pyreports
+  commands:
+    - pip list | grep -E "{{ name }}\s+{{ version }}"
```

Comment:
This is probably not necessary.

Reply:
Do you mean the grep test? I must have changed it for some reason... Although I can change back to pip check instead

Reply:
Okay, I've reverted it

Reply:
yes looks like running pip check fails the tests thats why I switched to grep

Reply:
oh wait, or maybe it's something else?